src/main.rs

```diff
@@ -182,6 +183,15 @@ struct Args {
         hide = true
     )]
     max_manifest_bucket_bytes: i64,
+    /// The storage type of nats streams, maps to [`StorageType`](async_nats::jetstream::stream::StorageType)
+    /// Possible values: "File" or "Memory", default is `File`
+    #[arg(
+        long = "stream-persistence",
+        env = "WADM_STREAM_PERSISTENCE",
+        default_value_t = StreamPersistence::File,
+        hide = true
```

Comment:
I think we should keep this un-hidden since it's a useful flag! (apologies if I said it should be hidden in the issue)